{{cookiecutter.repostory_name}}/nginx/templates/default.conf.template

```diff
@@ -134,3 +134,63 @@ server {
         proxy_pass http://app:8000/;
     }
 }
+
+{% if cookiecutter.monitoring == 'y' %}
+server {
+    server_name monitoring;
+    listen      10443           ssl http2;
+
+    ssl_session_cache         shared:SSL:20m;
+    ssl_session_timeout       10m;
+
+    ssl_protocols             TLSv1 TLSv1.1 TLSv1.2;
+    ssl_prefer_server_ciphers on;
+    ssl_ciphers               "ECDH+AESGCM:ECDH+AES256:ECDH+AES128:!ADH:!AECDH:!MD5;";
+
+    ssl_stapling              on;
+    ssl_stapling_verify       on;
+
+    ssl_certificate /etc/monitoring_certs/cert.crt;
+    ssl_certificate_key /etc/monitoring_certs/cert.key;
+    ssl_trusted_certificate   /etc/monitoring_certs/ca.crt;
+
+    ssl_client_certificate /etc/monitoring_certs/ca.crt;
+    ssl_verify_client on;
+
+    resolver 8.8.8.8 8.8.4.4 valid=300s;
+    resolver_timeout 5s;
+
+
+    gzip off;
```

Comment:
I think you can copy this from other parts of nginx config, or create some common config file and import it in both places